updated support phases to be described in a consistent way (#529)
suse-edge/suse-edge.github.io: asciidoc/edge-book/releasenotes.adoc

SUSE Edge is backed by award-winning support from SUSE, an established technology leader with a proven history of delivering enterprise-quality support services. For more information, see https://www.suse.com/lifecycle[https://www.suse.com/lifecycle] and the Support Policy page at https://www.suse.com/support/policy.html[https://www.suse.com/support/policy.html]. If you have any questions about raising a support case, how SUSE classifies severity levels, or the scope of support, please see the Technical Support Handbook at https://www.suse.com/support/handbook/[https://www.suse.com/support/handbook/].

> SUSE Edge "3.2" is supported for 18-months of production support, with an initial 6-months of GA\*, followed by 12-months of EOM\**. During the GA phase, SUSE may introduce new features (that do not break existing functionality), introduce bug fixes, and deliver security patches. During the EOM phase, only critical security and bug fixes will be introduced, with other fixes delivered at our discretion until it reaches its EOL\*** date.

Reviewer: I know the table is from Rancher lifecycle, but GA EOM and EOL are points in time not intervals, so it really feels wrong to speak of a GA phase and EOM phase.

Maybe just say something along the lines of:
```suggestion
SUSE Edge "3.2" is supported for 18-months of production support, with an initial 6-months from GA\* to EOM\**, followed by 12-months from EOM\** to EOL\***.
```

And I think it would be more convenient to have the table here, or at least provide a link to it.
